src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -177,7 +177,23 @@ class ServiceDetails extends React.Component<ServiceDetailsProps, ServiceDetails
                   <div>
                     Error Traces{' '}
                     <span>
-                      ({errorTraces}
+                      (
+                      {errorTraces !== -1 ? (
+                        errorTraces
+                      ) : (
+                        <OverlayTrigger
+                          overlay={
+                            <Tooltip id="error-trace-unknown-tooltip">
+                              There was a problem loading the <b>Error Trace</b> count from the server
+                            </Tooltip>
+                          }
+                          placement="top"
+                          trigger={['hover', 'focus']}
+                          rootClose={false}
+                        >
+                          <span>n/a</span>
+                        </OverlayTrigger>
+                      )}
                       {errorTraces > 0 && (
```

Comment:
Hi @josejulio , I was working in another PR to fix this that I forgot. when the value is -1 the Error traces tab should be a tab with name "Traces" ans show all the traces of this service. -1 means that there is a problem getting traces in Jaeger this could mean a timeout. When there is not JaegerURL configured the traces tab is hide.